adhocracy4/polls/models.py

```diff
@@ -164,7 +171,7 @@ def __str__(self):
 
     class Meta:
         ordering = ["id"]
-        unique_together = ("question", "creator")
+        unique_together = ("question", "creator", "content_id")
```

Comment:
does this work if either creator or content_id is None?

Reply:
no, None is treated as a normal value in the unique check as far as I understand, therefore e.g. `question1 + user1 + None` is a valid unique value. Same for `question1 + None + unique uuid` is a valid unique value. And both trigger an error if voted again with the same question and user/content_id